src/bastion/src/child_ref.rs

```diff
@@ -19,9 +19,28 @@ pub struct ChildRef {
     sender: Sender,
     name: String,
     path: Arc<BastionPath>,
+    // True if the ChildRef references a child that will receive user defined messages.
+    // use `ChildRef::new_internal` to set it to false, for internal use children,
+    // such as the heartbeat children for example
+    is_public: bool,
```

Comment:
Are we targeting master? I was sure this patch landed ^^'

Reply:
I believe, this patch came from master, because I mostly branching from master